RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -147,7 +147,9 @@
 eras.trackingPhase2PU140.toReplaceWith(detachedQuadStepTrajectoryFilterBase,
     _detachedQuadStepTrajectoryFilterBase.clone(
```

Comment:
This module is define in phase2Tracking140 era, but never really used in my opinion.